src/main/kotlin/fi/metatavu/edufication/pages/api/impl/V1ApiImpl.kt

```diff
@@ -0,0 +1,76 @@
+package fi.metatavu.edufication.pages.api.impl
+
+import fi.metatavu.edufication.pages.api.controller.PagesController
+import fi.metatavu.edufication.pages.api.model.Page
+import fi.metatavu.edufication.pages.api.spec.V1Api
+import org.hibernate.annotations.NotFound
+import org.jboss.resteasy.util.NoContent
+import java.util.*
+import javax.enterprise.context.RequestScoped
+import javax.inject.Inject
+import javax.ws.rs.BadRequestException
+import javax.ws.rs.core.Response
+
+@RequestScoped
+class V1ApiImpl: V1Api, AbstractApi()  {
+
+    @Inject
+    private lateinit var pagesController: PagesController
+
+    override fun createPage(page: Page?): Response {
```

Comment:
If page is required parameter in swagger, this can be defined as page: Page. The bean validation will take care of the validation